types/microsoft-graph/index.d.ts

```diff
@@ -12776,8 +12952,8 @@ export interface WindowsAppX extends MobileLobApp {
      */
     isBundle?: boolean;
     /**
-     * The value for the minimum applicable operating system. Valid values for a WindowsAppX app include v8_0, v8_1 and v10_0.
-     * If the app is a bundle, the minimum supported OS has to be at least v8_1.
+     * The value for the minimum applicable operating system. Valid values for a WindowsAppX app include v80, v81 and v100. If
```

Comment:
Here we have more instances where the usage of an _ in a string as a placeholder for a version delimiter is dropped, this feels off.